docs/userGuide/components/dynamic/quiz.md

```diff
@@ -0,0 +1,12 @@
+<question has-input>
+  Q1. Name all different types of voliatile memory that exists inside a typical household computer system.
+  <div slot="hint"></div>
```

Comment:
Should remove this slot since this is empty.

Reply:
Intentional to render the default `No hint is provided for this question.`